editions/tw5.com/tiddlers/filters/examples/substitute Operator (Examples).tid

```diff
@@ -6,7 +6,7 @@ type: text/vnd.tiddlywiki
 
 \define time() morning
 \define field() modified
-\define sentence() This tiddler was last $(field)$ on ${[{!!modified}format:date[DDth MMM YYYY]]}$
+\procedure sentence() This tiddler was last $(field)$ on ${[{!!modified}format:date[DDth MMM YYYY]]}$
```

Comment:
Perhaps you mean `<<field>>`

Reply:
@kookma no, the variable reference is intentional to show how it is further processed for variable substitution.